policyengine_us/tests/policy/baseline/gov/states/nj/tax/income/credits/eitc/nj_eitc_eligible.yaml

```diff
@@ -0,0 +1,41 @@
+- name: Individual without child that is still eligible for federal eitc based on age.
+  period: 2022
+  absolute_error_margin: 0
+  input:
+    people:
+      person1:
+        age: 30
+    households:
+      household:
+        members: [person1]
+        state_code: NJ
+  output:
+    nj_eitc_eligible: true
+
+- name: Individual without child that is not eligible for federal eitc based on age.
```

Comment:
```suggestion
- name: Filer without children that is ineligible for the federal EITC based on age, but still eligible for the NJ EITC because of lower age minimum.
```